packages/server/src/user/user.resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Mutation, Args, Resolver, Query } from '@nestjs/graphql';
+import { UserService } from './user.service';
+import { UserModel } from './model/user.model';
+import { User } from '@prisma/client';
+
+@Resolver()
+export class UserResolver {
+  constructor(private readonly userService: UserService) {}
+
+  @Query(() => UserModel)
+  async listUsers(): Promise<UserModel[]> {
+    return this.userService.findAllUsers();
+  }
+
+  @Query(() => UserModel)
+  getUser(@Args('id') id: string): Promise<UserModel> {
```

Comment:
in your client you say they type is ID here it is a string, add ID type here
```suggestion
  getUser(@Args('id', { type: () => ID}) id: string): Promise<UserModel> {
```